metadata.rb

```diff
@@ -4,7 +4,7 @@
 license 'Apache-2.0'
 description 'Installs/Configures stackstorm'
 long_description 'Installs/Configures stackstorm'
-version '0.7.0'
+version '0.7.1'
```

Comment:
```suggestion
version '0.7.0'
```
per previous comment